code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -586,22 +585,19 @@ butcher
 		var/mob/living/critter/flock/drone/F = holder.owner
 		usr = F // don't ask, please, don't
 		if(F?.set_hand(1)) // grip tool
-			if(!F.is_in_hands(container_target)) //if it's not in any of our hands, pick it up
-				F.drop_item() // drop whatever we're holding
```

Comment:
useful comment

Reply:
Idk, think drop_item() already explains this?